src/NuGet.Core/NuGet.Packaging/Signing/Native/ManagedCmsWrapper.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
+using NuGet.Common;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && IS_CORECLR
+    internal sealed class ManagedCmsWrapper : ICms
+    {
+        private readonly SignedCms _signedCms;
+
+        public ManagedCmsWrapper(SignedCms signedCms)
+        {
+            _signedCms = signedCms;
+        }
+
+        public byte[] GetPrimarySignatureSignatureValue()
+        {
+            if (_signedCms.SignerInfos.Count == 0)
```

Comment:
All of these checks should be `Count != 1` not `Count == 0` to be consistent with the error message.

Reply:
Thanks! Fixed.